src/utilities/utility.ts

```diff
@@ -129,6 +131,93 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
     return commandExecutionResults;
 }
 
+export async function getFilePathsConfigs(): Promise<any> {
+
+    let filePathsConfig: any = {};
+    const MANIFEST_PATHS_KEY = 'manifestFilePaths';
+    const HELM_CHART_KEY = 'helmChartFilePaths';
+    const DOCKERFILE_PATH_KEY = 'dockerfilePaths';
+    const DOCKERFILE_PATH_LABEL_KEY = 'dockerfile-path';
+
+    let inputManifestFiles = inputParams.manifests || [];
+    filePathsConfig[MANIFEST_PATHS_KEY] = inputManifestFiles;
+
+    let helmChartPaths = (process.env.HELM_CHART_PATHS && process.env.HELM_CHART_PATHS.split('\n').filter(path => path != "")) || [];

```

Comment:
Where is this value coming from? `k8s-bake`?

Reply:
No it is an environment variable in the workflow, the path to the helmchart, that is also used by the ks8-bake action.

Reply:
Did you callout these assumptions explicitly in the README?

Reply:
Yes, it is mentioned in the README.